### ISSUE TITLE
Support production SDK

### ISSUE THREAD

**Issue by GINAGC** [k8-proxy/go-k8s-process#118]
- Add production SDK to the minio version.  https://github.com/k8-proxy/go-k8s-process

**Comment by Lestat06** [comment]
@GINAGC this will be tested when we merge into main and create the AMIs

**Comment by GINAGC** [comment]
Blocked by https://github.com/k8-proxy/cs-k8s-api/issues/527
